classes/Mail.php

```diff
@@ -583,7 +583,11 @@ function ($carry, $item) {
                 'message' => &$message,
             ]);
 
-            $send = $swift->send($message);
+            if (defined('_PS_IN_TEST_') && _PS_IN_TEST_) {
```

Comment:
Sorry but 🤢 
https://github.com/PrestaShop/PrestaShop/blob/124376c915b4cd9b6d10db6777438feaa724c458/classes/Mail.php#L209
You can disable mail method otherwise...

Reply:
disabling mail method will do an early return
In that case I want Mail class to check all it can ("to" field and other stuff)
What do you propose for that ?

Reply:
Find a way to stug ` $send = $swift->send($message);`, Adding code for tests is never a good idea :/

Reply:
indead, but since the service is not injected it is hard to apply guidelines ;)